gui/qt/transaction_dialog.py

```diff
@@ -351,9 +351,7 @@ def format_amount(amt):
                 prevout_n = x.get('prevout_n')
                 cursor.insertText(prevout_hash[0:8] + '...', ext)
                 cursor.insertText(prevout_hash[-8:] + ":%-4d " % prevout_n, ext)
-                addr = x['address']
-                if isinstance(addr, PublicKey):
-                    addr = addr.toAddress()
```

Comment:
Yeah this is broken. Ha. In my commit for the asynch inputs dl I also got rid of this thing.  Surprised nothing ever hit this codepath...